packages/docs/PropTables/ModalPropTable.tsx

```diff
@@ -4,11 +4,8 @@ import { Code, PropTable } from '../components';
 
 export const ModalPropTable: React.FC = () => (
   <PropTable>
-    <PropTable.Prop name="isOpen" types="boolean" required>
-      Determines if the modal/dialog is open.
-    </PropTable.Prop>
-    <PropTable.Prop name="onClose" types="() => void" required>
-      Function that will be called on close events.
+    <PropTable.Prop name="actions" types="React.ReactNode">
+      Sets the actions of the modal.
```

Comment:
Maybe we should reword this to what _should_ be passed into the actions (i.e. `Button`'s).